hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
I used aiohttp 2.0.0 in my fist runs. The output for v2.2.0 and Python3.5 is quite the same:
https://gist.github.com/das7pad/e8f63175377576a0cb3466d20a417fa5#file-results-md

A first glance on the change from aiohttp 2.0.0 to 2.2.0:
The  session used in ``aiohttp.request`` is attached to the ``aiohttp.ClientResponse`` but not closed by anyone - only a direct access to the protected member ``aiohttp.ClientResponse._session.close()`` can close it.
Introduced in https://github.com/aio-libs/aiohttp/issues/1985
This is causing the resource warning even on the closed response.

``release`` or ``close`` : I am not familiar with connection pooling, but ``release`` should be fine.
In addition we could use a ``aiohttp.ClientSession`` attached to the ``hangups.client.Client`` and use it for all the aiohttp requests.

Using ``aiohttp==2.2.0`` and hangups spamms every log atm.